packages/text-field/src/TextField/TextField.tsx

```diff
@@ -0,0 +1,135 @@
+import { useComposedRefs } from "@telegraph/compose-refs";
+import { Icon } from "@telegraph/icon";
+import { clsx } from "clsx";
+import React from "react";
+
```

Comment:
Personally find this component name confusing vs `Field`. i.e. This component does not render a label or a message if i'm understanding it correctly. 

Feels like it should be called `TextInput`?

Reply:
Can change the name, was just following the naming convention in Figma.

Reply:
I strongly disagree with that statement. I think the right answer here is to change the name and make it consistent across both code and figma. But deviating on naming is going to lead to confusion later.